.github/workflows/build_and_test.yml

```diff
@@ -1204,3 +1204,48 @@ jobs:
 
             time aws s3 cp --only-show-errors s3://${BUCKET}/${S3_KEY} s3://${BUCKET}/${PREFIX}/${FILENAME}
           done
+
+  compute-node-image-merged-base:
+    needs: [ check-permissions, build-buildtools-image, tag ]
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [ v14, v15, v16 ]
+    defaults:
+      run:
+        shell: sh -eu {0}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+          fetch-depth: 0
+
+      - name: Build merged image base
+        run:
+          docker image build . -f Dockerfile.compute-node-simple -t compute-node-merged-base-${{ matrix.version }}
+            --build-arg PG_VERSION=${{ matrix.version }}
+
+  compute-node-image-merged:
+    needs: [ compute-node-image-merged-base ]
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: sh -eu {0}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+          fetch-depth: 0
+
+      - name: Build compute_ctl image
+        run:
+          docker image build . -f Dockerfile.compute-node-tools -t compute-node-merged-tools
+
+      - name: Build merged image
+        run:
+          docker image build . -f Dockerfile.compute-node-merged -t compute-node-merged
```

Comment:
I guess we need to release combine image with same release tag as existing image, so we can add build_args `${build_tag}` and image tag `compute-node:${release_tag}`

Reply:
Example:

```
docker build -f Dockerfile.compute-node-merged --build-arg BUILD_TAG=${{needs.tag.outputs.build-tag}} -t compute-node:${{needs.tag.outputs.build-tag}}
```